cmd/ethrex/initializers.rs

```diff
@@ -423,29 +421,29 @@ pub async fn init_l1(
         init_metrics(&opts, tracker.clone());
     }
 
-    cfg_if::cfg_if! {
-        if #[cfg(feature = "dev")] {
-            init_dev_network(&opts, &store, tracker.clone()).await;
-        } else {
-            if opts.p2p_enabled {
-                init_network(
-                    &opts,
-                    &network,
-                    &data_dir,
-                    local_p2p_node,
-                    local_node_record.clone(),
-                    signer,
-                    peer_table.clone(),
-                    store.clone(),
-                    tracker.clone(),
-                    blockchain.clone(),
-                    None
-                )
-                .await;
-            } else {
-                info!("P2P is disabled");
-            }
-        }
+    #[cfg(feature = "dev")]
+    if opts.dev {
+        init_dev_network(&opts, &store, tracker.clone()).await;
     }
+
+    if !opts.dev && opts.p2p_enabled {
```

Comment:
Can we merge these two ifs?

```suggestion
    if opts.dev {
        #[cfg(feature = "dev")]
        init_dev_network(&opts, &store, tracker.clone()).await;
     } else if opts.p2p_enabled {
```